tests/jetty-home-tester/src/main/java/org/eclipse/jetty/tests/hometester/JettyHomeTester.java

```diff
@@ -587,8 +586,8 @@ public boolean awaitLogsFileFor(Path logFile, String txt, long time, TimeUnit un
             thread.start();
             try
             {
-                long start = NanoTime.now();
-                while (NanoTime.since(start) < unit.toNanos(time))
+                long end = System.currentTimeMillis() + unit.toMillis(time);
```

Comment:
/me thinking loud :)
I always thought we could have very much simple code here by using this very convenient tool https://github.com/awaitility/awaitility